src/locales/en.json

```diff
@@ -83,6 +83,7 @@
     "person_risk_profile_header": "Risk Profile",
     "household_members_note": "If you are modeling exposure from members of your household, only count their contacts outside the house to avoid double-counting",
     "large_group_warning": "Warning: This is a large number of people. Remember, you only need to include the number of people who are within 15 feet of you (not everyone present in the area). However, gathering a large number of people could put everyone at risk and creates the possibility of superspreader event.",
+    "frontline_worker_acknowledgement": "We deeply appreciate the sacrifices that essential workers are making during the pandemic. We encourage you to find safe ways of supporting these people, keeping them socially included and cared for. They should be significantly safer once vaccinated.",
```

Comment:
Should this be: "... the sacrifices that **frontline** workers are making.."

Seems to me that that term is more clear and matches with the "healthcare / social worker" label.

Reply:
I updated this because I think it's more clear. Feel free to change back if you feel strongly about it.